app/models/nearby_facility.rb

```diff
@@ -7,13 +7,47 @@ class << self
     attr_writer :validate_on_load
 
     def query(params)
-      json_from_file = File.read(Rails.root.join('modules', 'va_facilities', 'nearby.json'))
-      mocked_response = JSON.parse(json_from_file)
-      return mocked_response if params[:street_address] && params[:city] && params[:state] && params[:zip]
+      if params[:street_address] && params[:city] && params[:state] && params[:zip]
+        isochrone_response = request_isochrone(params)
+        return get_facilities_in_isochrone(params, isochrone_response)
+      end
 
       NearbyFacility.none
     end
 
+    def request_isochrone(params)
+      params[:drive_time] = '30' unless params[:drive_time]
+      address = params[:street_address] + ' ' + params[:city] + ' ' + params[:state] + ' ' + params[:zip]
```

Comment:
not a huge deal but string interpolation saves objects vs concatenation in ruby
`"#{params[:street_address]  params[:city]  params[:state]  params[:zip]}"`

Reply:
Oh yeah, I should have known that.